docs/content/readme.md

```diff
@@ -47,7 +47,8 @@ See the [Getting Started setup guide](direct/get-started.md) for getting started
 
 ## Contributing
 
-We ❤️ our contributors! If you're interested in helping us out, please head over to our [Contributing](Contribution%20guidelines/CONTRIBUTING.md) guide.
+
+We ❤️ our contributors! If you're interested in helping us out, please head over to our [Contributing](contribution-guidelines/contributing-inc.md) guide.
```

Comment:
There is still a bit of organizational work to do, we have two website pages whose names/title sound very similar.

```console
mspreitz@mjs13 kubestellar % find docs/content -name "*ontrib*.md"
docs/content/direct/contribute.md
docs/content/contribution-guidelines/contributing-inc.md
...
```

But that's not for this PR.
@KPRoche